docs/modules/components/pages/processors/ollama_chat.adoc

```diff
@@ -0,0 +1,123 @@
+= ollama_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+
+////
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes, edit the corresponding source file under:
+
+     https://github.com/redpanda-data/connect/tree/main/internal/impl/<provider>.
+
+     And:
+
+     https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/plugin.adoc.tmpl
+////
+
+
+component_type_dropdown::[]
+
+
+Processor that uses the Ollama API to generate text.
+
+Introduced in version 4.31.0.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common config fields, showing default values
+label: ""
+ollama_chat:
+  server_address: "" # No default (optional)
```

Comment:
Writing `http://127.0.0.1:11434` is different because then the processor will not start a locally running ollama instance but use an already running server. Any ideas on how to make that clearer?

Reply:
You could say this?

The address of the Ollama server to use. The processor starts and runs a local Ollama server unless you specify the address of a local or remote server.

Reply:
Done!